src/extension.ts

```diff
@@ -584,6 +602,39 @@ function launchMetals(
     } else {
       outputChannel.appendLine("Debugging Scala sources is not supported");
     }
+    if (features.decorationProvider) {
+      client.onNotification(DecorationTypeDidChange.type, options => {
+        decorationType = window.createTextEditorDecorationType(options);
+      });
+      client.onNotification(DecorationsRangesDidChange.type, params => {
+        outputChannel.appendLine(JSON.stringify(params));
+        const uris = window.visibleTextEditors.map(e => e.document.uri);
+        outputChannel.appendLine(JSON.stringify(uris));
+        const editor = window.activeTextEditor;
+        if (editor) {
+          outputChannel.appendLine(
+            JSON.stringify(editor.document.uri.toString())
+          );
+        }
+        if (editor && params.uri == editor.document.uri.toString()) {
```

Comment:
```suggestion
        if (editor && params.uri === editor.document.uri.toString()) {
```

I've never found a valid use-case for `==` in JS

Reply:
Thanks, fixed.